### PR TITLE
use rollup alias to simplify imports

### PR DESCRIPTION
**Pull request by Schparky**
- configure @rollup/plugin-alias
- remove relative paths on `data` and `components` imports
- sort by import path
- separate internal imports from external imports by a blank line
- remove unused imports

### CODE REVIEW

**Review by captaincrazybro — APPROVED** [review]

File: rollup.config.js [hunk 1]
```diff
@@ -22,6 +31,7 @@ export default {
 	preserveEntrySignatures: 'strict',
 	inlineDynamicImports: true,
 	plugins: [
+    aliases,
```
Comment (hobbitronics) [inline rollup.config.js:34]: not sure what is going on with the indentation of this line.

File: src/components/App.svelte [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-import { init as loadEvents } from '../data/events'
-import { init as loadMessaging } from '../data/messaging'
-import { init as loadRequests } from '../data/requests'
-import { Router } from '@roxi/routify'
+import { init as loadEvents } from 'data/events.js'
+import { init as loadMessaging } from 'data/messaging.js'
+import { init as loadRequests } from 'data/requests.js'
+import { me } from 'data/user.js'
 import { routes } from '../../.routify/routes'
+
```
Comment (hobbitronics) [inline src/components/App.svelte:7]: is this line break intentional?

Comment (Schparky) [reply src/components/App.svelte:7]: Yes, as I mentioned in the PR description, it separates the internal imports from the external imports

File: src/components/ConversationListEntry.svelte [hunk 3]
```diff
@@ -1,10 +1,11 @@
 <script>
-import { me } from '../data/user'
+import CountIndicator from 'components/CountIndicator.svelte'
+import { listOtherParticipants, unreads } from 'data/messaging.js'
+import { me } from 'data/user.js'
+
```
Comment (hobbitronics) [inline src/components/ConversationListEntry.svelte:5]: same as comment above

File: rollup.config.js (hunk 1 above)
Comment (Schparky) [reply rollup.config.js:34]: Seems that much of the file is using tabs. I will convert to spaces to be consistent with the rest of the project.

File: src/components/App.svelte (hunk 2 above)
Comment (Schparky) [reply src/components/App.svelte:7]: Though if you don't like it, I would be OK with not doing this.

File: src/components/Error.svelte [hunk 4]
```diff
@@ -1,15 +1,15 @@
 <script>
-import { error, reset } from '../data/error'
+import { error, reset } from 'data/error.js'
```
Comment (hobbitronics) [inline src/components/Error.svelte:2]: just curious if .js is required in aliases. Also does vscode's autocomplete know how to use them?

Comment (Schparky) [reply src/components/Error.svelte:2]: Yes, the file extension is required. I found a github issue that seemed to explain the situation. There was a debate about it, and the final decision was that it was a bad idea to not require the file extension. Seems it's a bit of a controversy and the side that likes the extensions obviously won the debate.

Just tested vscode and it seems to not recognize the aliases. Probably would require an update to the Svelte plugin or a separate plugin altogether.

File: src/pages/requests/[requestId]/conversation/[conversationId]/index.svelte [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <script>
-import Request from '../../../../../components/Request.svelte'
```
Comment (hobbitronics) [inline src/pages/requests/[requestId]/conversation/[conversationId]/index.svelte:2]: this is where this change really shines

**Review by hobbitronics — APPROVED** [review]
Good changes! the less ./../.. the better

File: src/pages/requests/[requestId]/conversation/[conversationId]/index.svelte (hunk 5 above)
Comment (Schparky) [reply src/pages/requests/[requestId]/conversation/[conversationId]/index.svelte:2]: Right?! Also when copying or moving code from one level of hierarchy to another.

**Pull request merged**